package.json

```diff
@@ -17,15 +17,16 @@
     "release-notes",
     "changelog"
   ],
-  "dependencies": {
-    "@semantic-release/error": "4.0.0",
-    "@semantic-release/release-notes-generator": "^13.0.0",
-    "aggregate-error": "^5.0.0"
-  },
   "scripts": {
     "prettier": "prettier --write src/**/*.js"
   },
   "devDependencies": {
-    "prettier": "^3.2.5"
+    "prettier": "^3.2.5",
+    "@semantic-release/release-notes-generator": "^14.0.1",
+    "@semantic-release/error": "4.0.0",
+    "aggregate-error": "^5.0.0"
+  },
+  "peerDependencies": {
+    "semantic-release": "^21.0.0"
```

Comment:
is it correct to place on peerDependencies also @semantic-release/release-notes-generator, @semantic-release/error and aggregate-error ?